bin/build-plugin-zip.sh

```diff
@@ -81,6 +81,8 @@ status "Generating build..."
 npm run build
 status "Generating translation messages..."
 npm run gettext-strings
+status "Generating PHP file for wordpress.org to parse translations..."
+npm run pot-to-php
```

Comment:
Few lines below, we need to include this new PHP file in the generated ZIP archive.